src/neo/SmartContract/NefFile.cs

```diff
@@ -37,36 +36,32 @@ public class NefFile : ISerializable
         public Version Version { get; set; }
 
         /// <summary>
-        /// Script Hash
+        /// Script
         /// </summary>
-        public UInt160 ScriptHash { get; set; }
+        public byte[] Script { get; set; }
 
         /// <summary>
         /// Checksum
         /// </summary>
         public uint CheckSum { get; set; }
 
-        /// <summary>
-        /// Script
-        /// </summary>
-        public byte[] Script { get; set; }
+        public const int MaxScriptLength = 1024 * 1024;
```

Comment:
I'm not sure that should it be smaller then 1 MB?

Reply:
We can remove it and check `PayloadMaxSize`